lib/raidenclient/types.ts

```diff
@@ -45,8 +45,10 @@ export type ChannelEvent = {
 /**
  * The response for tokenPayment call.
  */
-export type TokenPaymentResponse = {
-  secret: string;
+export type TokenPaymentResponse = TokenPaymentRequest & {
+  secret?: string;
+  initiator_address?: string;
+  hashalgo?: string;
```

Comment:
Are we going to use these 2 new fields? If not it might be better to leave them out of the codebase for simplicity's sake.

Reply:
You're right, it doesn't make sense to keep them. Updated.